### PR TITLE
Revert "Create new meeting instead of using PMI (#147)"

### PR DESCRIPTION
**Pull request by mickmister**
This reverts commit 65af542cb1f059ea43085e29c29d0a536a14df6a.

#### Summary

This feature is partially holding up the next release, as it currently has functional and cosmetic issues. If we revert this PR, we just need to get https://github.com/mattermost/mattermost-plugin-zoom/pull/253 merged then we can have release testing.

https://community.mattermost.com/core/pl/poxkp98g9bgdzkxnniomd671gr for context

### CODE REVIEW

**Comment by larkox** [comment]
@mickmister There seem to be too many changes in the documentation, not sure how related. Are these intended?

**Comment by mickmister** [comment]
@larkox There are doc changes in the original PR https://github.com/mattermost/mattermost-plugin-zoom/pull/147/commits/5b1d61e5001de2731a7c1223c79508ae3f84968a, captured in the cherry pick commit above. This PR should be an accurate revert of the intended pieces from the original PR now

File: server/zoom/meeting.go [hunk 1]
```diff
@@ -3,39 +3,24 @@
...
-	Topic             string      `json:"topic"`
-	Type              MeetingType `json:"type"`
-	Status            string      `json:"status"`
-	StartTime         string      `json:"start_time"`
-	Duration          int         `json:"duration"`
-	Timezone          string      `json:"timezone"`
-	CreatedAt         string      `json:"created_at"`
-	Agenda            string      `json:"agenda"`
-	JoinURL           string      `json:"join_url"`
-	StartURL          string      `json:"start_url"`
```
Comment (larkox) [inline server/zoom/meeting.go:34]: Probably nothing important, but i noticed you "lost" this attribute (`StartURL`)

File: webapp/src/components/post_type_zoom/post_type_zoom.jsx [hunk 2]
```diff
@@ -102,7 +102,7 @@ export default class PostTypeZoom extends React.PureComponent {
             if (props.meeting_personal) {
                 subtitle = (
                     <span>
-                        {'Personal Meeting ID (PMI): '}
+                        {'Personal Meeting ID (PMI) : '}
```
Comment (larkox) [inline webapp/src/components/post_type_zoom/post_type_zoom.jsx:105]: This change probably can stay, depending on how clean you want to make the revert..

**Review by larkox — APPROVED** [review]
LGTM, just two minor things to keep in mind.

**Pull request closed without merging**